wrapper/smriprep_docker.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/env python
+"""
+The sMRIPrep on Docker wrapper
+
+
+This is a lightweight Python wrapper to run sMRIPrep.
+Docker must be installed and running. This can be checked
+running ::
+
+  docker info
+
+Please report any feedback to our GitHub repository
+(https://github.com/poldracklab/smriprep) and do not
+forget to credit all the authors of software that sMRIPrep
+uses (https://smriprep.readthedocs.io/en/latest/citing.html).
+"""
+import sys
+import os
+import re
+import subprocess
+from warnings import warn
+
+__version__ = '99.99.99'
+__copyright__ = 'Copyright 2019, Center for Reproducible Neuroscience, Stanford University'
+__credits__ = ['Craig Moodie', 'Ross Blair', 'Oscar Esteban', 'Chris Gorgolewski',
+               'Shoshana Berleant', 'Christopher J. Markiewicz', 'Russell A. Poldrack']
```

Comment:
I did not update this...

Reply:
Why not?

Reply:
Um, mostly because I forgot, then made a note to myself, then forgot again.